ui/src/actions/actions-ui.js

```diff
@@ -12,6 +19,15 @@ export const appsTableSort = (index, direction) => {
   };
 };
 
+export const appDetailsSort = (index) => {
+  return {
+    type: APP_DETAILS_SORT,
+    payload: {
+      index: index
```

Comment:
if want to persist the sort for both tables we need to use a different sortBy state for each rather than resetting this one.

Reply:
Are we persisting the sort? (just confirming)

Reply:
Going to resolve this for now. If persist on sort is required it can be addressed in another PR. Going to try close this PR as it's blocking others.